### DISCUSSION TITLE
Question: Custom popup menu questions

### DISCUSSION THREAD

**Discussion by chaudry-786** [neoclide/coc.nvim#3970] [Q&A] [answered]
## Result from CocInfo

vim version: NVIM v0.7.2
node version: v16.13.1
coc.nvim version: 0.0.82-eaf1c634 2022-08-01 18:13:40 +0800
coc.nvim directory: /home/sabah/.vim/plugged/coc.nvim
term: tmux
platform: linux

Hi I have got 2 questions about custom Popup menu;

1) Can it show in writing what type of suggestion it is? e.g "Class", "Function"
Desired result
https://preview.redd.it/ivw962yjd3f91.png?width=911&format=png&auto=webp&s=c3585530d0111baef551ab565cf1a5d9d5715958

2) What highlight group controls matched text colour. Currently for me, it's blue but I would like to change it to another colour.

![image](https://user-images.githubusercontent.com/73726199/182203288-88f262c3-98e5-4216-88dd-0c167c6df9df.png)

At last, big thank you for your work. really appreciated.

**Comment by chemzqm** [comment, marked as answer]
`:h coc-config-suggest-completionItemKindLabels`
`:h CocMenuSel`

**Comment by chaudry-786** [comment]
Thank you, question 2 is solved. 

Regarding question 1, I'm aware of suggest.completionItemKindLabels; my question was can text be added too along with Label? (to specify whether it's Class, Function, Keyword etc

**Reply by chemzqm** [reply]
Yes, you can.

**Reply by chaudry-786** [reply]
Please tell me how? 
![Screenshot from 2022-08-01 20-37-46](https://user-images.githubusercontent.com/73726199/182231876-f2d47661-6f1f-4607-b1ca-de7de82be29f.jpg)

**Reply by chemzqm** [reply]
something like
```json
  "suggest.completionItemKindLabels": {
    "function": "function",
  },
```

**Reply by chaudry-786** [reply]
oh wow.. thank you. 

I tried adding it before but I got "String is longer than the maximum length of 1." so I stuck to Icon only.

But thank you, despite this warning it shows what I expect to see.
